packages/plans-grid/src/mixins.scss

```diff
@@ -0,0 +1,139 @@
+	line-height: 40px;
+}
+
+@mixin onboarding-heading-text-mobile {
+	@include onboarding-font-recoleta;
+	font-size: 32px;
+	line-height: 40px;
+}
+
+@mixin onboarding-large-text {
+	font-size: 16px;
+	line-height: 24px;
+}
+
+@mixin onboarding-medium-text {
+	font-size: 14px;
+	line-height: 17px;
+}
+
+@mixin onboarding-small-text {
+	font-size: 12px;
+	line-height: 14px;
+}
+
+@mixin onboarding-x-small-text {
+	font-size: 11px;
+	line-height: 13px;
+}
+
+@mixin onboarding-block-margin {
+	margin: 0 $gutenboarding-block-margin-mobile;
+
+	@include break-small {
+		margin: 0 $gutenboarding-block-margin-small;
+	}
+
+	@include break-medium {
+		margin: 0 $gutenboarding-block-margin-medium;
+	}
+}
+
+@mixin onboarding-heading-padding {
+	margin: $gutenboarding-heading-padding-mobile;
+
+	@include break-mobile {
+		margin: $gutenboarding-heading-padding-desktop;
+	}
+}
+
+/* (1920x1080) Full HD Display */
+@mixin onboarding-break-gigantic() {
+	@media ( min-width: 1920px ) {
+		@content;
+	}
+}
+
+@mixin onboarding-break-mobile-tall() {
+	@media ( min-height: 750px ) {
+		@content;
+	}
+}
+
+@mixin onboarding-block-edge2edge-container {
+	width: calc( 100% + #{$gutenboarding-block-margin-mobile * 2} );
+	margin-left: #{$gutenboarding-block-margin-mobile * -1};
+	margin-right: #{$gutenboarding-block-margin-mobile * -1};
+
+	@include break-small {
+		width: calc( 100% + #{$gutenboarding-block-margin-small * 2} );
+		margin-left: #{$gutenboarding-block-margin-small * -1};
+		margin-right: #{$gutenboarding-block-margin-small * -1};
+	}
+
+	@include break-medium {
+		width: calc( 100% + #{$gutenboarding-block-margin-medium * 2} );
+		margin-left: #{$gutenboarding-block-margin-medium * -1};
+		margin-right: #{$gutenboarding-block-margin-medium * -1};
+	}
+}
+
+@mixin onboarding-block-edge2edge-content {
+	border-left: $gutenboarding-block-margin-mobile solid transparent;
+	border-right: $gutenboarding-block-margin-mobile solid transparent;
+
+	@include break-small {
+		border-left: $gutenboarding-block-margin-small solid transparent;
+		border-right: $gutenboarding-block-margin-small solid transparent;
+	}
+
+	@include break-medium {
+		border-left: $gutenboarding-block-margin-medium solid transparent;
+		border-right: $gutenboarding-block-margin-medium solid transparent;
+	}
+}
+
+@mixin onboarding-block-edge2edge-columns {
+	&:first-child {
+		border-left: $gutenboarding-block-margin-mobile solid transparent;
+	}
+	&:last-child {
+		border-right: $gutenboarding-block-margin-mobile solid transparent;
+	}
+
+	@include break-small {
+		&:first-child {
+			border-left: $gutenboarding-block-margin-small solid transparent;
+		}
+		&:last-child {
+			border-right: $gutenboarding-block-margin-small solid transparent;
+		}
+	}
+
+	@include break-medium {
+		&:first-child {
+			border-left: $gutenboarding-block-margin-medium solid transparent;
+		}
+		&:last-child {
+			border-right: $gutenboarding-block-margin-medium solid transparent;
+		}
+	}
+}
```

Comment:
I wonder what would be a good shared place for these mixins. 🤔 New package? `@automattic/components`?

Reply:
A new package with these assets would be good to have. We have also the [variables](https://github.com/Automattic/wp-calypso/pull/42764/files#diff-d65534841d3b05f200b095dba551ab5a) to share between different packages and Gutenboarding.